src/base/rss/rss_autodownloader.cpp

```diff
@@ -108,8 +108,6 @@ AutoDownloader::AutoDownloader()
     m_instance = this;
 
     m_fileStorage = new AsyncFileStorage(specialFolderLocation(SpecialFolder::Config) / Path(CONF_FOLDER_NAME));
-    if (!m_fileStorage)
```

Comment:
@glassez 
Do you expect to catch this exception, or let the exception propagate out is fine?

Reply:
This is the case when the application cannot continue to work correctly. Maybe we should just rethrow it as a RuntimeError with a more appropriate message? IIRC, such errors have some handling in `Application::exec()` just before exit.

Reply:
>Maybe we should just rethrow it as a RuntimeError with a more appropriate message? 

I don't know, is RSS considered as a critical component of the application? If not, then terminate the app would not be acceptable.

Reply:
> If not, then terminating the app on such error would not be acceptable.

What do you suggest instead?
I don't think the application will be able to work stably even if some non-critical component is faulty. Maybe just turn RSS off? But in this case, the user must be explicitly notified of this, and this is not so easy to do in the general case.

>is RSS considered as a critical component of the application?

What can we consider an "uncritical" component? Any component used, I suppose, is critical for the user, isn't it?

Reply:
>What can we consider an "uncritical" component? Any component used, I suppose, is critical for the user, isn't it?

I suppose so. User did explicitly enabled RSS, so he would expect it to work.
OK, let's keep it as-is.